components/backends/sglang/deploy/disagg_planner.yaml

```
@@ -71,14 +71,16 @@ spec:
+            - /bin/sh
+            - -c
           args:
-            - python
-            - -m
-            - planner_sla
-            - --environment=kubernetes
-            - --backend=sglang
-            - --adjustment-interval=60
-            - --profile-results-dir=/workspace/profiling_results
+            - >-
+              python3 -m planner_sla 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)